tests/foreman/cli/test_ansible.py

```diff
@@ -334,6 +335,35 @@ def test_positive_run_concurrent_jobs(self, rex_contenthosts, target_sat):
         target_sat.cli.GlobalParameter().delete({'name': param_name})
         assert len(target_sat.cli.GlobalParameter().list({'search': param_name})) == 0
 
+    @pytest.mark.parametrize('value', [0, -2, 2.5, 'a', '2.5'])
```

Comment:
mind adding `ids` with some sane names to each value?
i'd rather have test called `test_negative_invalid_concurrency_level_string` and `test_negative_invalid_concurrency_level_float` than `test_negative_invalid_concurrency_level_a` and `test_negative_invalid_concurrency_level_2.5`